src/foam/nanos/crunch/connection/CapabilityPayloadDAO.js

```diff
@@ -260,12 +260,18 @@ foam.CLASS({
 
       try {
         CapabilityPayload receivingCapPayload = (CapabilityPayload) obj;
+        var pm1 = PM.create(x, true, "receivingCapPayload");
         Map<String,FObject> capabilityDataObjects = (Map<String,FObject>) receivingCapPayload.getCapabilityDataObjects();
+        pm1.log(x);
 
+        var pm2 = PM.create(x, true, "getGrantPath");
         List grantPath = ((CrunchService) x.get("crunchService")).getGrantPath(x, receivingCapPayload.getId());
+        pm2.log(x);
         processCapabilityList(x, grantPath, capabilityDataObjects);
 
+        var pm3 = PM.create(x, true, "find_");
```

Comment:
This is not very descriptive, the pminfo table will simply show 'find_'.  Please add at least the class name to these.